examples/kafka/src/main/scala/org/apache/gearpump/streaming/examples/kafka/wordcount/KafkaWordCount.scala

```diff
@@ -59,6 +62,9 @@ object KafkaWordCount extends App with ArgumentsParser {
 
   val config = parse(args)
   val context = ClientContext(config.getString("master"))
+
+  implicit val system = context.system
```

Comment:
Further improve this,

add doc link to akka
```
+  /**
+   * This will de-serialize value to object instance
+   *
+   * To do de-serialization, this requires an implicit ActorSystem, as
+   * the ActorRef and possibly other akka classes deserialization
+   * requires an implicit ActorSystem.
+   *
+   * @see [[http://doc.akka.io/docs/akka/snapshot/scala/serialization.html#A_Word_About_Java_Serialization]]
+   */
```